src/conditions/check_fields.rs

```diff
@@ -232,7 +232,14 @@ fn build_predicate(
         "eq" | "equals" => EqualsPredicate::new(target, arg),
         "neq" | "not_equals" => NotEqualsPredicate::new(target, arg),
         "contains" => ContainsPredicate::new(target, arg),
-        "prefix" => PrefixPredicate::new(target, arg),
+        "prefix" => {
+            warn!(
+                message = "The \"prefix\" comparison predicate is deprecated, use \"starts_with\" instead",
```

Comment:
Should this be rate limited?

Reply:
I don't think so. AFAICT this only triggers when the config is parsed, so it would only happen once per offending condition in the configs.